client.go

```diff
@@ -44,7 +49,24 @@ import (
 	"golang.org/x/oauth2/clientcredentials"
 )
 
-const defaultStorageURL = "https://api.e3db.com"
+const (
+	defaultStorageURL                   = "https://api.e3db.com"
+	SECRET_UUID                         = "38bb737a-4ce0-5ead-8585-e13ea23b09a6"
+	SECRET_WRITER_USERNAME_METADATA_KEY = "username"
+	SECRET_SHARED_METADATA_KEY          = "shared"
+	SECRET_FILE_SIZE_METADATA_KEY       = "size"
+	SECRET_FILENAME_METADATA_KEY        = "fileName"
+	SECRET_TYPE_METADATA_KEY            = "secretType"
+	SECRET_DESCRIPTION_METADATA_KEY     = "description"
+	SECRET_NAME_METADATA_KEY            = "secretName"
+	SECRET_VERSION_METADATA_KEY         = "version"
+	SECRET_VALUE_DATA_KEY               = "secretValue"
+)
+
+var (
+	encryptedFileName = "encrypted"
```

Comment:
is this used by the sdk or only for tests? If the latter can you move it into a test file, if the former can you move it up to constants, and call it something more intention revealing like `DefaultEncryptedFileName`